Dockerfile

```diff
@@ -42,8 +42,6 @@ ARG AIRFLOW_HOME=/opt/airflow
 ARG AIRFLOW_UID="50000"
 ARG AIRFLOW_GID="50000"
 
-ARG CASS_DRIVER_BUILD_CONCURRENCY="8"
```

Comment:
I kept this for "future" proofing, as cassandra-driver wasn't yet publishing py 3.9 wheels.

Reply:
Ah. True. Will restore it back. We might soon add python 3.9 finally (waiting for newest beam release).

Reply:
I restored it only for CI image as it only makes sense there. For Prod build we still want to build cassandra driver with cython if not already available, and I thin the build_concurrency, I seriously doubt someone will care too much about the potential few minutes improvements there. the variable is undocumented anyway and I am not sure it is valid still in the latest cassandra driver.